src/Charts/Gauge/Gauge.js

```diff
@@ -36,6 +36,9 @@ class Gauge extends Component {
             gauge: {
                 fullCircle: true,
                 label: {
+                    format: function () {
```

Comment:
We don't need this function, do we?

Reply:
this removes the value in the center of the gauge whereas the `show: false` removes the min/max labels in the gauge. We want to remove the value in the center of the gauge in favor of creating our own data in the center so that we have more control of it.